docs/mobile/exercises/README.md

```diff
@@ -0,0 +1,284 @@
+critical pedagogical information here. If you want to put in a message of
+encouragement or jubilation however, feel free!
+
+## feedback_wrong
+
+```yaml
+feedback_wrong: "Don't forget that R is 1-indexed, not 0-indexed like Python."
+```
+
+The `feedback_wrong` field is the fallback feedback message when a particular
+distractor hasn't been supplied with one. Usually, you should be giving specific
+feedback for each individual distractor, because specific, enlightening feedback
+messages are one of the most effective ways of teaching on DataCamp for
+Mobile. If you find that the same feedback message is appropriate for multiple
+distrators, however, this field is provided for convenience.
+
+## tabs
+
+```yaml
+tabs:
+  - key: people
+    type: table
+```
+
+Tabs appear in the top right of the exercise and serve as a way to display
+datasets or files. The appropriate asset must be defined in the [course
+manifest](../repo-structure.md). A tab field has two subfields: `key`, which is
+the alias for the asset as defined in the manifest, and `type`, which specifies
+how to render the asset. Currently only `file` and `table` are supported.
+
+# Summary of available blocks
+
+| Block             | Reorder | Tap   | Select Code | Select Output | Select Table | Multiple Choice |
+|-------------------+---------+-------+-------------+---------------+--------------+-----------------|
+| context           | :ok:    | :ok:  | :ok:        | :ok:          | :ok:         | :ok:            |
+| question          | :ok:    | :ok:  | :ok:        | :ok:          | :ok:         | :ok:            |
+| key               | :req:   | :req: | :req:       | :req:         | :req:        | :req:           |
+| code              | :req:   | :req: | :req:       | :ok:          | :ok:         | :ok:            |
+| output            | :ok:    | :ok:  | :ok:        | :req:         | :ok:         | :ok:            |
+| distractor_blanks | :x:     | :req: | :x:         | :x:           | :x:          | :x:             |
+| distractor_code   | :x:     | :x:   | :req:       | :x:           | :x:          | :x:             |
+| distractor_output | :x:     | :x:   | :x:         | :req:         | :x:          | :x:             |
+| distractor_table  | :x:     | :x:   | :x:         | :x:           | :req:        | :x:             |
+| distractor_text   | :x:     | :x:   | :x:         | :x:           | :x:          | :req:           |
+| feedback          | :ok:    | :ok:  | :ok:        | :ok:          | :ok:         | :ok:            |
+| feedback_wrong    | :ok:    | :ok:  | :ok:        | :ok:          | :ok:         | :ok:            |
+
+# YAML tips and tricks
+
+You will often be writing code blocks or output blocks that span multiple
+lines. You *can* use explicit newlines `\n`, but the easier and more readable
+option is to use a feature of the YAML spec known as **literal scalar
+blocks**. Just put `|-` after the semicolon `;` of a field and then indent the
```

Comment:
semicolon? Shouldn't it be a normal colon `:`?

Reply:
haha yes. I must have had a brain aneurism.